subsys/bluetooth/controller/ll_sw/ull_iso_internal.h

```diff
@@ -4,21 +4,42 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#if defined(CONFIG_BT_CTLR_ADV_ISO) || defined(CONFIG_BT_CTLR_CONN_ISO)
 /* Calculate ISO PDU buffers required considering SDU fragmentation */
-/* FIXME: Calculation considering both Connected and Broadcast ISO PDU
- *        fragmentation.
+#if defined(CONFIG_BT_CTLR_ADV_ISO) || defined(CONFIG_BT_CTLR_CONN_ISO)
+/* Internal ISO Tx SDU maximum length.
+ * A length that is minimum of the resultant combination of the HCI ISO data fragments provided and
+ * the user configured maximum transmit SDU length (CONFIG_BT_CTLR_ISO_TX_SDU_LEN_MAX).
+ */
+#define BT_CTLR_ISO_TX_SDU_LEN_MAX MIN(((CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE * \
+					 CONFIG_BT_CTLR_ISO_TX_BUFFERS) - \
+					BT_HCI_ISO_SDU_HDR_SIZE), \
+				       CONFIG_BT_CTLR_ISO_TX_SDU_LEN_MAX)
+
+BUILD_ASSERT(BT_CTLR_ISO_TX_SDU_LEN_MAX == CONFIG_BT_CTLR_ISO_TX_SDU_LEN_MAX,
+	     "Insufficient ISO Buffer Size and Count for the required ISO SDU Length");
```

Comment:
Shouldn't this be
```suggestion
BUILD_ASSERT(BT_CTLR_ISO_TX_SDU_LEN_MAX >= CONFIG_BT_CTLR_ISO_TX_SDU_LEN_MAX,
	     "Insufficient ISO Buffer Size and Count for the required ISO SDU Length");
```

For example `CONFIG_BT_CTLR_ISO_TX_SDU_LEN_MAX` can be 100 and `BT_CTLR_ISO_TX_SDU_LEN_MAX` can be 200, which should be valid. 

Since `BT_CTLR_ISO_TX_SDU_LEN_MAX` is minimum `CONFIG_BT_CTLR_ISO_TX_SDU_LEN_MAX` as per the above macro, what does this assert actually check?

Reply:
`BT_CTLR_ISO_TX_SDU_LEN_MAX` is minimum of `200` and `100`   in line 13, hence the build_assert checks for `==`. `BT_CTLR_ISO_TX_SDU_LEN_MAX`  will not be greater than `CONFIG_BT_CTLR_ISO_TX_SDU_LEN_MAX` (application required max. SDU length).